src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,18 @@
+import Styled from "@emotion/styled";
+import { palette } from "@leafygreen-ui/palette";
+import { size } from "constants/tokens";
+
+const { gray, white } = palette;
+
+const NavBar: React.FC = () => <Container>OUR TRUSTY NAVBAR</Container>;
+
+const Container = Styled.div`
```

Comment:
I know this is just a fake navbar, but why not use semantic html 😁 
```suggestion
const Container = Styled.nav`
```